src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
+const tree = document.querySelector('.tree');
+
+for (const li of tree.querySelectorAll('li')) {
+  const span = document.createElement('span');
+
+  li.prepend(span);
+  span.append(span.nextSibling);
```

Comment:
Appending `span.nextSibling` to `span` is incorrect because `span` has just been created and doesn't have a next sibling yet. You might want to append the existing content of `li` to `span` instead. Consider using `span.append(...li.childNodes)` to move the existing children of `li` into `span`.